src/openscm_runner/adapters/fair2/_run_fair.py

```diff
@@ -0,0 +1,366 @@
+                                                    RF_df.loc["HFC143a"].to_numpy() +\
+                                                    RF_df.loc["HFC227ea"].to_numpy() +\
+                                                    RF_df.loc["HFC245fa"].to_numpy() +\
+                                                    RF_df.loc["HFC4310mee"].to_numpy() +\
+                                                    RF_df.loc["SF6"].to_numpy()
+    data["Effective Radiative Forcing|Montreal Protocol Halogen Gases"] =   RF_df.loc["CFC11"].to_numpy() +\
+                                                                            RF_df.loc["CFC12"].to_numpy() +\
+                                                                            RF_df.loc["CFC113"].to_numpy() +\
+                                                                            RF_df.loc["CFC114"].to_numpy() +\
+                                                                            RF_df.loc["CFC115"].to_numpy() +\
+                                                                            RF_df.loc["CCl4"].to_numpy() +\
+                                                                            RF_df.loc["CH3CCl3"].to_numpy() +\
+                                                                            RF_df.loc["HCFC22"].to_numpy() +\
+                                                                            RF_df.loc["HCFC141b"].to_numpy() +\
+                                                                            RF_df.loc["HCFC142b"].to_numpy() +\
+                                                                            RF_df.loc["Halon1211"].to_numpy() +\
+                                                                            RF_df.loc["Halon1202"].to_numpy() +\
+                                                                            RF_df.loc["Halon1301"].to_numpy() +\
+                                                                            RF_df.loc["Halon2402"].to_numpy() +\
+                                                                            RF_df.loc["CH3Br"].to_numpy() +\
+                                                                            RF_df.loc["CH3Cl"].to_numpy()
+    #data["Effective Radiative Forcing|Aerosols|Direct Effect"] = RF_df.loc["CO2"].to_numpy()
+    #data["Effective Radiative Forcing|Aerosols"] = RF_df.loc["CO2"].to_numpy()
+
+    data["Surface Temperature"] = T_df.to_numpy()
+    data["Surface Temperature (GMST)"] = T_df.to_numpy() * factors["gmst"]
+    #data["Airborne Fraction"] = airborne_emissions
+    #data["Effective Climate Feedback"] = lambda_eff
+    #data["Heat Content"] = ohc
+    #data["Heat Content|Ocean"] = ohc * factors["ohu"]
+    #data["Net Energy Imbalance"] = heatflux
+    #data["Heat Uptake"] = heatflux * toa_to_joule
+    #data["Heat Uptake|Ocean"] = heatflux * toa_to_joule * factors["ohu"]
+
+    unit["Atmospheric Concentrations|CO2"] = "ppm"
```

Comment:
as above, there must be a cleaner way to do this unit handling rather than hard-coding here (I was hoping units would be included in FaIR2.0's pandas interface)